src/opower/opower.py

```diff
@@ -272,6 +274,7 @@ async def async_get_forecast(self) -> list[Forecast]:
                             utility_account_id=str(
                                 forecast["preferredUtilityAccountId"]
                             ),
+                            service_point_id=forecast["servicePointId"],

```

Comment:
There is no such field for my PG&E account in the combined-forecast endpoint.

Reply:
Hmm, good to know.  I was just assuming it was everywhere, but looks like the forecasts use a different structure.

I can make the field optional and just not populate it in the forecast.  That said, I was mainly looking to add this because it allowed me to use it as a differentiator in the HA integration.  If we're going the route of using UUID instead, then this isn't necessary.

I'll update this PR with making it optional, but its a lower priority change now.